.github/workflows/mutest-issue-generate.yml

```diff
@@ -14,23 +14,23 @@ jobs:
       -
         name: Run mutation test
         continue-on-error: true
-        run: make test-mutation
+        run: make test-mutation MODULES=tokenfactory
```

Comment:
nit: set the env directly instead

Reply:
Can you explain further what you mean here? I though I would need to be explicit with what I want tested here since it is a github action

Reply:
Github supports an `env` field.

```yml
env:
  MODULES: tokenfactory
```

https://docs.github.com/en/actions/learn-github-actions/environment-variables

Reply:
Learn something new everyday, thanks!